packages/auto-label/src/auto-label.ts

```diff
@@ -261,9 +174,14 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
       if (foundAPIName) {
         logger.info('The label already exists on this issue');
       } else {
-        await handler.addLabels(github, owner, repo, issueNumber, [
-          githubLabel,
-        ]);
+        await github.issues
+          .addLabels({
+            owner,
+            repo,
+            issue_number: issueNumber,
+            labels: [githubLabel],
+          })
+          .catch(logger.error);
         logger.info(
           `Label added to ${owner}/${repo} for issue ${issueNumber} is ${githubLabel}`
         );
```

Comment:
Should this just throw the error, rather than log the error then say, "label added"? Same below.

Reply:
Probably.  This was supposed to be a behavior consistent refactor, so I kept the behavior.  Will come back around and fix up a few of these odds and ends.